modular/translations/code/translation_data/ru_names.toml

```
@@ -23480,3 +23470,2255 @@ dative = "конструкционному узлу"
+
+["HUDS (CHOOSE 1)"]
+nominative = "ЭКРАНЫ (ВЫБЕРИТЕ 1)"
+
+["BELTS (CHOOSE 1)"]
+nominative = "ПОЯС (ВЫБЕРИТЕ 1)"
+
+["G8-A General Utility Pouch"]
+nominative = "Поясная разгрузка G8-A"
+
+["M276 Toolbelt Rig (Full)"]
+nominative = "Пояс для инструментов"
+
+["Military Police Belt"]
+nominative = "Пояс военной полиции"
+
+["M276 Medical Storage Rig"]
+nominative = "Поясная разгрузка санитара"
+
+["M276 Medical Lifesaver Rig"]
+nominative = "Спасательная сумка санитара"
+
+["M276 Ammo Load Rig"]
+nominative = "Пояс для боеприпасов"
+
+["M276 Holster Toolrig"]
+nominative = "Пояс с кобурой для инструментов"
+
+["M276 M82F Holster Rig"]
+nominative = "Пояс с кобурой для ракетницы M82F"
+
+["POUCHES (CHOOSE 2)"]
+nominative = "ПОДСУМКИ (ВЫБЕРИТЕ 2)"
+
+["Medium General Pouch"]
+nominative = "Подсумок общего назначения"
+
+["Construction Pouch"]
+nominative = "Подсумок строителя"
+
+["Electronics Pouch (Full)"]
+nominative = "Подсумок электрика"
+
+["Tools Pouch (Full)"]
+nominative = "Подсумок для инструментов"
+
+["Electronics Pouch (Full)"]
+nominative = "Подсумок для осветительных огней"
+
+["First-Aid Pouch (Refillable Injectors)"]
+nominative = "Медицинский подсумок (инжекторы)"
+
+["First-Aid Pouch (Splints, Gauze, Ointment)"]
+nominative = "Медицинский подсумок (шины, бинты, мазь)"
+
+["First-Aid Pouch (Pill Packets)"]
+nominative = "Медицинский подсумок (таблетки)"
+
+["Autoinjector Pouch"]
+nominative = "Подсумок для инжекторов"
+
+["Medical Kit Pouch"]
+nominative = "Подсумок с набором санитара"
+
+["Large General Pouch"]
+nominative = "Большой подсумок общего назначения"
+
+["Large Pistol Magazine Pouch"]
+nominative = "Большой подсумок для пистолетных магазинов"
+
+["Large Magazine Pouch"]
+nominative = "Большой подсумок для магазинов"
+
+["Sidearm Pouch"]
+nominative = "Кобура для пистолета"
+
+["Large Shotgun Shell Pouch"]
+nominative = "Большой подсумок для дробовика"
+
+["Tools Pouch (Full)"]
+nominative = "Подсумок для инструментов"
+
+["MASK (CHOOSE 1)"]
+nominative = "МАСКИ (ВЫБЕРИТЕ 1)"
+
+["Gas Mask"]
+nominative = "Противогаз"
+
+["Heat Absorbent Coif"]
+nominative = "Инфракрасная балаклава"
+
+["Rebreather"]
+nominative = "Респиратор"
+
+["ColMarTech Commanding Officer Equipment Rack"]
+nominative = "Раздатчик экипировки командующего офицера «КолМорТех»"
+
+["ColMarTech Maintenance Technician Equipment Rack"]
+nominative = "Раздатчик экипировки техника «КолМорТех»"
+
+["Map"]
+nominative = "Карта"
+
+["Headset"]
+nominative = "Гарнитура"
+
+["Insulated Gloves"]
+nominative = "Резиновые перчатки"
+
+["ARMOR (CHOOSE 1)"]
+nominative = "БРОНЯ (ВЫБЕРИТЕ 1)"
+
+["UNIFORM (CHOOSE 1)"]
+nominative = "УНИФОРМА (ВЫБЕРИТЕ 1)"
+
+["HELMET (CHOOSE 1)"]
+nominative = "ШЛЕМЫ (ВЫБЕРИТЕ 1)"
+
+["BACKPACK (CHOOSE 1)"]
+nominative = "РЮКЗАКИ (ВЫБЕРИТЕ 1)"
+
+["BELT (CHOOSE 1)"]
+nominative = "ПОЯС (ВЫБЕРИТЕ 1)"
+
+["Technician Backpack"]
+nominative = "Рюкзак техника"
+
+["Technician Satchel"]
+nominative = "Ранец техника"
```

```suggestion
["technician satchel"]
nominative = "ранец техника"
gender = "male"
```
И если это перевод предмета, следует добавлять сразу же пол предмета